recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
     - missing-namespace-string-npos.patch
 
 build:
-  number: 4
+  number: 5
   # This package is a dependency of hdfs3, which is primarily used with HDFS on Linux.
   # There isn't libboost=1.73 and libgsasl=1.8 on s390x
```

Comment:
`libboost-1.82.0` already exists on `s390x` but `libgsasl >=1.8` doesn't exist so the comment should be modified.